pkgs/tools/networking/easyrsa/default.nix

```
@@ -2,13 +2,13 @@
 
 stdenv.mkDerivation rec {
   pname = "easyrsa";
-  version = "3.1.7";
+  version = "3.1.8";
 
   src = fetchFromGitHub {
     owner = "OpenVPN";
     repo = "easy-rsa";
     rev = "v${version}";
-    sha256 = "sha256-zdVcT04nj7eE1a6M7WHeWpwG/TVTwyK+WgD70XwPXfY=";
+    sha256 = "zdVcT04nj7eE1a6M7WHeWpwG/TVTwyK+WgD70XwPXfY=";
```

```suggestion
    hash = "sha256-zdVcT04nj7eE1a6M7WHeWpwG/TVTwyK+WgD70XwPXfY=";
```
Please don't cripple hashes

---

That's even more important and I totally missed that

---

Ahhhh I totally missed this while testing something. Sorry all.

---

OK, yeah, what happened is I was trying to see if I could add PKCS#11 support to nixpkgs' easyrsa, and tested easyrsa 3.0.8, then realized I ought to update this in nixpkgs too. So version 3.1.8 does not actually exist, and 3.1.7 is the latest. Sorry for the confusion. :sweat_smile: